.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug report
+about: Create a report to help us improve
+---
+
+<!--
+NOTICE: While GitHub is the preferred channel for reporting issues/feedback, this is not a mechanism for receiving support under any agreement or SLA. If you require immediate assistance, please use official support channels.
+-->
+
+<!--

```

Comment:
We should probably take out verbiage here around the CLI, as it doesn't really apply to this project.